lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -221,9 +236,20 @@
         return resolvedLocale;
     }
 
-    // ECMA402 section 9.2.2 (#sec-bestavailablelocale)
-    // REVIEW(jahorto): We don't cache the result here because the only callsite is resolveLocaleLookup, which has its own cache
+    /**
+     * Determines the best possible locale available in the system
+     * 
+     * ECMA-402: #sec-bestavailablelocale
```

Comment:
Did you deliberately remove the reference to section 9.2.2?

Reply:
I have been told that the section numbers can change over time but the hash-links don't

Reply:
Yes, section numbers in the spec are unstable. The anchor tags are nearly completely stable unless removed entirely. This is what @bterlson recommends for referencing the spec (even if referencing a certain edition of the spec, as you can compare the section between versions if you want to see what changed).